rules/Removing/Rector/FuncCall/RemoveFuncCallRector.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Removing\Rector\FuncCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use Rector\Core\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Core\Rector\AbstractRector;
+use Rector\NodeTypeResolver\Node\AttributeKey;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use Webmozart\Assert\Assert;
+
+/**
+ * @see \Rector\Tests\Removing\Rector\FuncCall\RemoveFuncCallRector\RemoveFuncCallRectorTest
+ */
+final class RemoveFuncCallRector extends AbstractRector implements ConfigurableRectorInterface
+{
+    /**
+     * @var string[]
+     */
+    private array $removedFunctions = [];
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('Remove function', [
+            new ConfiguredCodeSample(
+                <<<'CODE_SAMPLE'
+$x = 'something';
+var_dump($x);
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+$x = 'something';
+CODE_SAMPLE
+                ,
+                ['var_dump']
+            ),
+        ]);
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [FuncCall::class];
+    }
+
+    /**
+     * @param FuncCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        foreach ($this->removedFunctions as $removedFunction) {
```

Comment:
```php
$expr = $node->expr;

if (! $expr instanceof FuncCall) {
    return null;
}

// then verify against $expr
```

Reply:
I tried this, but it failed in the case of an `if` condition. For a conditional, this check would be `$node->cond`.

Reply:
You can commit your try and we can see what error to fix